<launch>
  <arg name="team" default="1"/>
  
  <node pkg="player" name="pointcloud_node" type="pointcloud_node" output="screen">
    <param name="image_transport" value="compressed"/>
  </node>
   
</launch>
